fidesctl/src/fidesapi/main.py

```diff
@@ -33,11 +40,25 @@ async def health() -> Dict:
     return {"data": {"message": "Fides service is healthy!"}}
 
 
+@app.post("/admin/db/{action}", tags=["Admin"])
+async def db_action(action: DBActions) -> Dict:
```

Comment:
i think this is a bit overkill, i'd rather just have two separate endpoints. Was there a case you thought of where this was a better design? or did you try to design after based on the other CRUD endpoints? 

for return status i think a 204 would work, no need for an actual message imo (assuming its separate endpoints)

Reply:
thinking more on this, I'm ok with this as-is. still feels a little wonky to me but also don't feel the need to see it changed.